### DISCUSSION TITLE
How can I create tree like subsub...section?

### DISCUSSION THREAD

**Discussion by khajavi** [facebook/docusaurus#4494] [Q&A]
How many levels of subsections, docusaurus support?

Can I have subsubsection on sidebars?

**Comment by slorber** [comment]
Docusaurus 2 support as much levels as you want and your theme can render properly, so it's your responsibility to remain reasonable ;)

![image](https://user-images.githubusercontent.com/749374/112188022-e97c6900-8c02-11eb-99b5-de3620f1a84b.png)

```js
    {
      type: 'category',
      label: 'Docusaurus',
      items: [
        {
          type: 'category',
          label: 'Docusaurus',
          items: [
            {
              type: 'category',
              label: 'Docusaurus',
              items: ['introduction', 'design-principles', 'contributing'],
            },
          ],
        },
      ],
    },
```

**Reply by khajavi** [reply]
Nice! 
Thank you @slorber
